### PR TITLE
[shopsys] webpack fixes

### LINKED ISSUE #1613: Wrong behavior of images on ProductDetail in administration
### What is happening
SSFW version: monorepo 9.0.0-dev (d803fb5f4a41cde774f82a2a9ff7f2811170b2b8)
Page: Product detail in administration

Problem 1:  
- Product images cannot be reordered. Dragging product image by a cross no longer works.

Problem 2: 
- It's possible to upload other files than images (for example PDF). This file is shown in the images section (as pdf). 
- After sending the form with `Save changes` button trigger server-side form validation with general error (`Please check the correctness of all data filled.`) and `Image can be only in JPG, GIF or PNG format` near newly uploaded file.
- I remove the file, upload image instead (jpg) and I cannot `Save changes` due to JS validation (
`Image can be only in JPG, GIF or PNG format`
- I remove the file and still cannot save changes due to the same error.

Not so big problem 3: 
- When I upload the image, I can set the name. But this name is not used anywhere, and image is still renamed with its ID. Fields with no added value are confusing and should not be present.

### Expected result

1. It's possible to reorder images again.
1. I can upload images anytime and cannot upload anything else.
1. Field with name is no longer rendered.

### LINKED ISSUE #1621: DatePicker is not rendered properly
### What is happening
Datepicker (used for example in administration on product edit page on field "Selling start date")
is missing some styles (probably). 

Datepicker is not usable
Snímek obrazovky 2020-01-26 v 23 18 15

_may be related with webpack in #1545?_

 

### Expected result
Datepicker is nice again

### PR DESCRIPTION
**Pull request by pesektomas**
| Q             | A
| ------------- | ---
|Description, reason for the PR| This PR fixed some bugs from #1545
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| closes #1613, closes #1621 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: upgrade/UPGRADE-v9.0.0-dev.md [hunk 1]
```diff
@@ -619,6 +619,24 @@ There you can find links to upgrade notes for other versions too.
...
+        });
+        ```
+    - update your `templates/Front/Content/Search/searchBox.html.twig`
+        ```diff
+        -  <button type="submit" class="btn search__form__button">
+        +  <button type="submit" class="btn search__form__button js-search-autocomplete-submit">
+               {{ 'Search [verb]'|trans }}
+           </button>
+        ```
+
```
Comment (grossmannmartin) [inline upgrade/UPGRADE-v9.0.0-dev.md]: I was intrigued you introduced a new section for these fixes. 
Is any reason to not add these instructions to already existing webpack upgrades?

File: packages/framework/src/Resources/styles/admin/main.less [hunk 2]
```diff
@@ -43,6 +43,9 @@
 @import "libs/tooltip/tooltip-custom.less";
 @import "libs/animation.less";
 
+@import "libs/jquery/theme.less";
+@import "libs/jquery/datepicker.less";
```
Comment (grossmannmartin) [inline packages/framework/src/Resources/styles/admin/main.less]: Does it make sense to do a code review from frontend-developers for introduced less files? 
TBH I'm not sure if all the rules are necessary or not.

Comment (pesektomas) [reply packages/framework/src/Resources/styles/admin/main.less]: I consulted this amendment with mr. Gott. What do you thing with `all the rules`?

Comment (grossmannmartin) [reply packages/framework/src/Resources/styles/admin/main.less]: CSS selectors and declarations in `theme.less` and `datepicker.less`. 
My question is mainly _"Do we need/want the whole files like they are?"_

File: upgrade/UPGRADE-v9.0.0-dev.md (hunk 1 above)
Comment (pesektomas) [reply upgrade/UPGRADE-v9.0.0-dev.md]: Hmm, I don't know. This instructions unrelated with webpack for me (sorry for PR name). But  if you want I will move it.

File: packages/framework/src/Resources/styles/admin/main.less (hunk 2 above)
Comment (grossmannmartin) [reply packages/framework/src/Resources/styles/admin/main.less]: Resolved personally, files are ok

**Pull request merged**